daemon/endpoint.go

```diff
@@ -814,19 +814,19 @@ func (d *Daemon) OnIPIdentityCacheGC() {
 				ipcache.IPIdentityCache.RLock()
 				defer ipcache.IPIdentityCache.RUnlock()
 
-				keysToRemove := map[ipCacheBPF.Key]struct{}{}
+				keysToRemove := map[string]*ipCacheBPF.Key{}
```

Comment:
This had to change because we need to de-dup and the pointer won't do that, right? It used the internal contents (and so the IP) before?

Reply:
Correct